docs/modules/ROOT/pages/type-definitions/database-mapping.adoc

```diff
@@ -43,3 +42,94 @@ interface UserLivesInProperties @relationshipProperties {
     since: DateTime @alias(property: "moveInDate")
 }
 ----
+
+[[type-definitions-node]]
+== `@node`
+
+The `@node` directive is used to specify the configuration of a GraphQL object type which represents a Neo4j node.
+
+=== Definition
+
+[source, graphql, indent=0]
+----
+"""Indicates the underlying Neo4j labels that this type will be mapped to."""
+directive @node(label: String, additionalLabels: [String]) on OBJECT
```

Comment:
This still needs to be married up to the definition in "packages/graphql/src/schema/validation/directives.ts":

```suggestion
"""Informs @neo4j/graphql of node metadata"""
directive @node(
    """Map the GraphQL type to a custom Neo4j node label"""
    label: String
    """Map the GraphQL type to match additional Neo4j node labels"""
    additionalLabels: [String]
) on OBJECT
```